File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task140_GEMINI_gemini-1.5-pro-001.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter the first number: ");
        double num1 = scanner.nextDouble();
        
        System.out.print("Enter the operator (+, -, *, /): ");
        String op = scanner.next();
        
        System.out.print("Enter the second number: ");
        double num2 = scanner.nextDouble();
        
        double result;
        
        switch (op) {
            case "+":
                result = num1 + num2;
                System.out.println(result);
                break;
            case "-":
                result = num1 - num2;
                System.out.println(result);
                break;
            case "*":
                result = num1 * num2;
                System.out.println(result);
                break;
            case "/":
                if (num2 == 0) {
                    System.out.println("Division by zero not allowed");
                } else {
                    result = num1 / num2;
                    System.out.println(result);
                }
                break;
            default:
                System.out.println("Invalid input");
        }
    }
}

File: dataset/output/GEMINI_gemini-1.5-pro-001/Task140_GEMINI_gemini-1.5-pro-001.c
#include <stdio.h>

int main() {
    double num1, num2;
    char op;

    printf("Enter the first number: ");
    scanf("%lf", &num1);
    printf("Enter the operator (+, -, *, /): ");
    scanf(" %c", &op);
    printf("Enter the second number: ");
    scanf("%lf", &num2);

    switch (op) {
        case '+':
            printf("%lf
", num1 + num2);
            break;
        case '-':
            printf("%lf
", num1 - num2);
            break;
        case '*':
            printf("%lf
", num1 * num2);
            break;
        case '/':
            if (num2 == 0) {
                printf("Division by zero not allowed
");
            } else {
                printf("%lf
", num1 / num2);
            }
            break;
        default:
            printf("Invalid input
");
    }

    return 0;
}